packages/noports_core/lib/src/sshnp/impl/sshnp_openssh_local_impl.dart

```diff
@@ -66,6 +67,10 @@ class SshnpOpensshLocalImpl extends SshnpCore
 
     logger.info('Sending request to sshnpd');
 
+    final server = await ServerSocket.bind(InternetAddress.anyIPv4, 0);
+    int localRvPort = server.port;
+    await server.close();
```

Comment:
Above code is to identify a port that is available for the rv process to bind to